src/services/transaction/TransactionFeeEstimateService.ts

```diff
@@ -32,9 +32,20 @@ export class TransactionFeeEstimateService extends AbstractService {
 		transaction: string
 	): Promise<RuntimeDispatchInfo> {
 		const { api } = this;
+		const apiAt = await api.at(hash);
```

Comment:
for my understanding, what is the difference of pulling the hash at this point and not inside the `try`?

Reply:
No big difference, more so for readability. The `try` is really just dealing with the async call whereas establishing the `api.at()` is declaring the runtime api.